src/components/Todo/TodoStyles.ts

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+import { Grid, Text } from "@mantine/core";
+
+export const StyledText = styled(Text)``;
+
+export const TodoWrapper = styled(Grid)`
+  width: 100%;
+  height: 4rem;
+  flex-wrap: nowrap;
+
+  &.mantine-Grid-root .mantine-Col-root {
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    margin: 0;
+
+    .mantine-Text-root {
```

Comment:
V jedné komponentě (Grid) styluješ jinou (Text). To není dobrý nápad i když tady chápu tu motivaci. Ale jak na to koukám, říkám si, že text todo by měl být asi vidět celý, ne?